.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -10,12 +10,15 @@ assignees: ''
 Please take the time to describe the bug in detail, so that other contributors can pick it up and implement or declare it done/obsolete in case it was fixed out of band.
 
 **What is wrong**
-Please describe what is wrong with the system as of writing: what is the expected state of things, and what is the actual state of things.
+Describe what is wrong with the system as of writing: what is the expected state of things, and what is the actual state of things.
 If this is a bug reproducible by a user, provide instructions to reproduce.
 If applicable, add screenshots, screencasts, or video to help explain your problem.
 
 **How do you know it is wrong**
-Please link any documents, Slack threads, quote personal communication, reference your personal domain expertise etc. explaining why you think the current state of things is bad and the expected state of things is good.
+Link any documents, Slack threads, quote personal communication, reference your personal domain expertise etc. explaining why you think the current state of things is bad and the expected state of things is good.
 
 **Code pointers**
-Please give some thoughts on roughly where in code do you think the necessary changes might be located.
+Give some thoughts on roughly where in code do you think the necessary changes might be located.
+
+**Other**
+Make sure to link the issue with a milestone, add appropriate labels, and mention dependent/blocked issues if appropriate.
```

Comment:
And honestly, "make sure" is also an empty phrase :)

Reply:
Removed.